pkg/neutronapi/volumes.go

```diff
@@ -39,25 +39,9 @@ func GetInitVolumeMounts(extraVol []neutronv1beta1.NeutronExtraVolMounts, svc []
 //	mechanism.
 func GetAPIVolumes(name string, extraVol []neutronv1beta1.NeutronExtraVolMounts, svc []storage.PropagationType) []corev1.Volume {
 	var scriptsVolumeDefaultMode int32 = 0755
-	var config0640AccessMode int32 = 0640
+	var config0644AccessMode int32 = 0644
 
 	res := []corev1.Volume{
-		{
-			Name: "etc-machine-id",
```

Comment:
These being dropped as part of https://github.com/openstack-k8s-operators/neutron-operator/pull/167